.php_cs

```diff
@@ -0,0 +1,33 @@
+<?php
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__ . "/src");
+
+return PhpCsFixer\Config::create()
+    ->setRules(
+        [
+            '@PSR2'                               => true,
+            'array_syntax'                        => [
+                'syntax' => 'short'
+            ],
+            'class_keyword_remove'                => false,
+            'concat_space'                        => [
+                'spacing' => 'one'
+            ],
+            'combine_consecutive_unsets'          => true,
+            'general_phpdoc_annotation_remove'    => [
+                'annotations' => ['@author'],
+            ],
+            'linebreak_after_opening_tag'         => true,
+            'no_short_echo_tag'                   => true,
+            'no_useless_else'                     => true,
```

Comment:
@cafferata @qurben I actually believe we should after all get rid of these extra `else` statements. They do look like noise to the most of the developer community, and the members of developer community are our most valuable contributors.

@andrederoos shall we switch to the modern code standard and get rid of all these redundant `else`'s? With short methods involving little branching (like what we have) it makes especially good sense. (Also, the guy on the video has his point!)